<launch>
    <include file="$(find px4_controller)/launch/px4_sitl.launch"/>
    
    <node pkg="joy" type="joy_node" name="joy_input" output="screen"/>
    <node pkg="px4_controller" type="translator.py" name="RPTYpub" output="screen"/>
</launch>
